diff -r -u liquid_feedback_frontend-v4.0.0-orig/env/util/login.lua liquid_feedback_frontend-v4.0.0/env/util/login.lua
--- liquid_feedback_frontend-v4.0.0-orig/env/util/login.lua	2021-04-28 12:38:36.000000000 +0200
+++ liquid_feedback_frontend-v4.0.0/env/util/login.lua	2023-07-20 14:26:50.844541403 +0200
@@ -74,9 +74,9 @@
     app.session.lang = member.lang
   end
 
-  if member.password_hash_needs_update then
-    member:set_password(password)
-  end
+  -- if member.password_hash_needs_update then
+  --   member:set_password(password)
+  -- end
   
   member:save()
   app.session.member = member
